.ci/docker/Dockerfile_dswx_hls

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021, by the California Institute of Technology.
+# Copyright 2022, by the California Institute of Technology.
```

Comment:
Generally speaking, I advise _against_ including the licensing, copyright, and authorship in every code file. It becomes cumbersome to maintain and also (especially for authorship) doesn't quite reflect all contributors to the code. 

My suggestion is to keep the LICENSE file in the repo up-to-date, and leave the authorship to Git commits (and Git blames)

Reply:
I was under the impression that, despite being an open source project, we were still responsible for including this information for all our production code. But I can't remember if it was in our requirements, or a JPL/NASA mandate though. Could use some clarification on this.

Reply:
@collinss-jpl - JPL's stance is to place copyrights in [a central license file](https://github.com/nasa/opera-sds-pge/blob/main/LICENSE.txt#L190) within OSS repos to make maintainability easier for developers. Not a huge deal and no one is going to ping you, but it'll probably save you having to change a code file for a non-code related legalize reason. I'll send you some more info privately.